docs/shared/migration/migration-angular.md

```diff
@@ -205,58 +204,50 @@ yarn create nx-workspace myorg --preset=angular
 
 When prompted for the `application name`, enter the _project name_ from your current `angular.json` file.
```

Comment:
Above this, it says to use `--preset angular`, it should now be `--preset angular-standalone`.

Or honestly.. maybe `--preset empty`?